src/checks_util.h

```diff
@@ -81,13 +81,17 @@ class ASTNodeRule {
       rule_;
 };
 
+// It will return the corresponding string of a range in a sql.
```

Comment:
Returns the sql substring corresponding to the given range.

Reply:
Erase GetName